src/customRules/trailingCommaInterfaceRule.ts

```diff
@@ -4,11 +4,15 @@ import * as Lint from 'tslint';
 class TrailingCommaInterfaceWalker extends Lint.RuleWalker {
    private static FAILURE_STRING = 'Trailing comma is required in interfaces properties.';
    public visitInterfaceDeclaration(node: ts.InterfaceDeclaration) {
+    const potentialDelimiters = [';'];
     node.members.map((propertySignature) => {
-        if (propertySignature.getText().slice(-1) !== ',') {
-            this.addFailure(this.createFailure(propertySignature.getStart(),
-                                               propertySignature.getWidth(),
-                                               TrailingCommaInterfaceWalker.FAILURE_STRING));
+        const lastChar = propertySignature.getText().slice(-1);
+        if (lastChar !== ',') {
```

Comment:
Try to make your callbacks do one thing well.  Conditions are cognitive load.

Chain a couple of callbacks instead:
```
    node.members
      .filter((propertySignature) => propertySignature.getText().slice(-1) !== ',')
      .forEach((propertySignature) => {
        const lastChar = propertySignature.getText().slice(-1);
```